docs/sources/panels-visualizations/visualizations/flame-graph/index.md

```diff
@@ -0,0 +1,90 @@
+
+Each block can have 0 or more siblings but can only have one parent.
+
+In figure 1 we have many rows of colored blocks but also have grayed out sections which represent a set of blocks that represent a relatively short execution time and so were collapsed together into one section for performance reasons.
+
+{{< figure src="/static/img/docs/flame-graph-panel/flame-graph-hovering.png" class="docs-image--no-shadow" max-width= "900px" caption="Figure 2 - Hovering over flame graph" >}}
+
+As there is more information to a block other than just its label, we decided to hide this information by default and show it on hover.
+
+Hovering over a specific block pops up a tooltip that shows you what the profile is measuring, in this case total time, along with other information such as this block's % of total time, the amount of time it took and the samples that were measured.
+
+{{< figure src="/static/img/docs/flame-graph-panel/flame-graph-clicking.png" class="docs-image--no-shadow" max-width= "900px" caption="Figure 3 - Clicking on a block" >}}
+
+As there may be multiple blocks per row, we may want to focus on a particular block in the row and its children only to further drill down the stack of function calls.
+
+Clicking on a particular block will do just that, hiding the block's siblings. The clicked block is set to 100% of the flame graphs width and all its children blocks are now shown with their widths updated relative to their new parents width.
+
+This process can be repeated to focus on finding the amount of resources that particular blocks and their children are consuming.
+
+{{< figure src="/static/img/docs/flame-graph-panel/flame-graph-search.png" class="docs-image--no-shadow" max-width= "900px" caption="Figure 4 - Search" >}}
+
+When you would like to quickly find the blocks with particular labels you can use the search field. Simply type in the label you want to search for and all of the blocks in the table with that text in their label will remain colored with all of the remaining blocks grayed out.
+
+You can also click on any block or hover over it to narrow your search.
+
+## Top table
+
+The top table allows you to visualize which symbols take up the most resources in your profile.
+
+The table has three columns, symbols, self and total - representing the self time and total time of each symbol. The table is sorted by self time by default, but can be reordered by total time or symbol name upon clicking the column headers.
+
+{{< figure src="/static/img/docs/flame-graph-panel/top-table.png" class="docs-image--no-shadow" max-width= "900px" caption="Figure 5 - Top table" >}}
+
+Clicking on a particular row of the top table will add that row's symbol to the search input which will search for that symbol in the flame graph.
+
+Clicking on that top table row again will remove its symbol from the search input.
```

Comment:
```suggestion

```

Reply:
I updated the text as this is useful info for the user.